addons/mail/static/tests/qunit_suite_tests/components/chat_window_manager_tests.js

```diff
@@ -928,7 +928,7 @@ QUnit.test('chat window: scroll conservation on toggle discuss', async function
 
     const pyEnv = await startServer();
     const mailChannelId1 = pyEnv['mail.channel'].create({});
-    for (let i = 0; i < 10; i++) {
+    for (let i = 0; i < 100; i++) {
```

Comment:
Test hard-checked scroll happens with precise scroll position.
It assumes there are enough messages to have at least a specific scroll height.

Before this PR, 10 messages were enough for `142`, but now it's too low (just does about `129`).

So I just changed it to `100`, so it's definitely enough for making a scroll change to `142`.